core-build/gulp-core-build/package.json

```diff
@@ -50,7 +50,9 @@
     "semver": "~5.3.0",
     "through2": "~2.0.1",
     "yargs": "~4.6.0",
-    "z-schema": "~3.18.3"
+    "z-schema": "~3.18.3",
+    "jest-cli": "~20.0.4",
+    "jest": "~20.0.4"
```

Comment:
Are both of these required? Or does jest-cli bring in jest?

Reply:
jest brings in jest-cli

Reply:
jest brings jest-cli with a ^ range. jest-cli brings in the world.

---
In reply to: [144161619](https://github.com/Microsoft/web-build-tools/pull/391#discussion_r144161619) [](ancestors = 144161619)